docs/source/user/installing.rst

```diff
@@ -235,6 +235,9 @@ vary with target operating system and hardware. The following lists them all
     :ref:`runtime type-checking <type_anno_check>`.
   * ``cuda-python`` - The NVIDIA CUDA Python bindings. See :ref:`cuda-bindings`.
     Numba requires Version 11.6 or greater.
+  * ``cubinlinker`` and ``ptxcompiler`` to support
+    :ref:`minor-version-compatibility`.
```

Comment:
I changed the title of the section to "CUDA Minor Version Compatibility" because I realised the title in itself wasn't clear, so this link now appears in the docs as being clear it's CUDA-related.

Reply:
Thanks.